internal/webhooks/tempostack_webhook.go

```diff
@@ -39,7 +39,7 @@ var (
 type TempoStackWebhook struct {
 }
 
-const defaultRouteGatewayTLSTermination = v1alpha1.TLSRouteTerminationTypePassthrough
+const defaultRouteGatewayTLSTermination = v1alpha1.TLSRouteTerminationTypeReencrypt
```

Comment:
Do we need a changelog for this?

Reply:
I think so, I mean is an small change, but may be the users wanted to be aware?